website/docs/reference/widgets/list.md

```diff
@@ -1,11 +1,418 @@
+---
+description: >-
+  List widget reference
+---
+
 # List
 
 The List widget provides a way to iterate over a structured dataset (array of objects) and display the data in sections that repeat vertically without writing any code. Each list item can contain other widgets to display data or capture user input.
 
+When you drag a List widget on the canvas, it has **Image** and **Text** widgets embedded by default. You can add more widgets on the first list item container. When you connect the dataset to the List and bind the data to each widget within the first list item, the widgets in the subsequent list items are updated with values from the dataset automatically. 
```

Comment:
Some parts of this para repeat information on the iterative dataset. We can exclude that.
(You can add widgets on the first list item container. When you connect the dataset to the List and bind the data to each widget within the first list item, the widgets in the subsequent list items are updated with values from the dataset automatically. )

 I'm not sure what value will this infor add when we are saying that the default widgets are Image and Text.